### PR TITLE
Define the configuration for print collectors

### PR DESCRIPTION
**Pull request by tareq89**
## Description

Link to the GitHub issue: https://github.com/opencrvs/opencrvs-core/issues/9201

Trying to achieve print certificate collector options by country config conditionals.

## Checklist

- [x] I have tested the changes locally, and written appropriate tests
- [ ] I have tested beyond the happy path (e.g. edge cases, failure paths)
- [ ] I have updated the changelog with this change (if applicable)
- [x] I have updated the GitHub issue status accordingly

### CODE REVIEW

File: src/form/v2/birth/forms/printForm/collectors.ts [hunk 1]
```diff
@@ -0,0 +1,363 @@
...
+      id: 'v2.event.birth.action.certificate.form.section.requester.label'
+    },
+    conditionals: [
+      {
+        type: ConditionalType.SHOW,
+        conditional: field('informant.relation').isEqualTo(InformantType.MOTHER)
+      }
+    ],
+    options: [
+      {
```
Comment (cibelius) [inline src/form/v2/birth/forms/printForm/collectors.ts:37]: If we want, we can save these options to variables which can be reused here, e.g.:

```
     const motherOption = {
        label: {
          id: 'v2.event.birth.action.certificate.form.section.requester.mother.label',
          defaultMessage: 'Print and issue to Informant (Mother)',
          description: 'This is the label for the field'
        },
        value: InformantType.MOTHER
      }
```

File: src/form/v2/birth/forms/printForm/index.ts [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+      id: 'collector.collect.payment',
+      type: PageTypes.enum.FORM,
+      title: {
+        id: 'event.birth.action.print.collectPayment',
+        defaultMessage: 'Collect Payment',
+        description: 'This is the title of the section'
+      },
+      fields: [
+        {
+          id: 'collector.collect.payment.data.after365',
```
Comment (cibelius) [inline src/form/v2/birth/forms/printForm/index.ts:95]: This could be renamed `collector.collect.payment.data.afterLateRegistration` or such, since the late reg configuration could be something else than 365 days

Sorry my comment in the previous PR was off 😅

File: src/form/v2/birth/index.ts [hunk 3]
```diff
@@ -25,7 +25,7 @@ import {
 import { advancedSearchBirth } from './advancedSearch'
 import { Event } from '@countryconfig/form/types/types'
 import { SCOPES } from '@opencrvs/toolkit/scopes'
-import { BIRTH_CERTIFICATE_COLLECTOR_FORM } from './forms/print-certificate'
+import { BIRTH_CERTIFICATE_COLLECTOR_FORM } from './forms/printForm/print-certificate'
```
Comment (cibelius) [inline src/form/v2/birth/index.ts:28]: This import seems wrong

**Review by cibelius — APPROVED** [review]

File: src/form/v2/birth/forms/printForm/collectors.ts [hunk 4]
```diff
@@ -0,0 +1,363 @@
...
+        label: {
+          id: 'v2.event.birth.action.certificate.form.section.requester.mother.label',
+          defaultMessage: 'Print and issue to Informant (Mother)',
+          description: 'This is the label for the field'
+        },
+        value: InformantType.MOTHER
+      },
+      {
```
Comment (cibelius) [inline src/form/v2/birth/forms/printForm/collectors.ts:45]: If we'd like, we could save the options which are used in many selects to variables, e.g.:



```
      const fatherOption = {
        label: {
          id: 'v2.event.birth.action.certificate.form.section.requester.father.label',
          defaultMessage: 'Print and issue to father',
          description: 'This is the label for the field'
        },
        value: InformantType.FATHER
      }
```
